SAKPaaS/src/app/components/search-bar/search-bar.component.html

```diff
@@ -1,9 +1,9 @@
 <form class="search-bar-form">
   <mat-form-field>
     <input matInput placeholder="{{ 'search-bar.placeholder' | translate }}" aria-label="Supermarkt Suchen"
```

Comment:
Please replace the hard-coded literal "Supermarkt Suchen" with a text-symbol.

Reply:
Ich hab das `aria-label` entfernt, da `<input>` das nicht unterstützt (wird durch placeholder beschrieben).

Hab aber `search-bar.placeholder`-Translation überarbeitet.